.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -4,12 +4,12 @@ about: Create a report to help us improve Electron
 
 ---
 
-* **Electron Version** (output of `node_modules/.bin/electron --version`):
-  * <!-- e.g. 4.0.3 -->
-* **Operating System** (Platform and Version):
-  * <!-- e.g. macOS 10.13.6 / Windows 10 (1803) / Ubuntu 18.04 x64 -->
-* **Last known working Electron version** (if applicable):
-  * <!-- e.g. 3.1.0 -->
+* **Electron Version:**
+<!-- (output of `node_modules/.bin/electron --version`) e.g. 4.0.3 -->
```

Comment:
The initial `*` here was to make it visually stand out to a human. Can we keep it? I like moving the `--version` hint inside the comment though.

Reply:
done!